pkg/images/conf.go

```diff
@@ -11,6 +11,9 @@ type ImgConf struct {
 	Parent string `json:"parent,omitempty"`
 	// ImageSize is the size of the image (defaults to images.DefaultImageSize)
 	ImageSize string `json:"image_size,omitempty"`
+	// Bootable indicates if the image should be bootable, i.e. contain a kernel
+	// and a bootloader.
+	Bootable *bool `json:"bootable,omitempty"`
```

Comment:
One thing that came to my mind: instead of adding this option and make the configuration more complicated (and also different between arm64 and amd64) maybe what could be done is still making:
- bootable by default on amd64
- non-bootable by default on arm64 (maybe change that in the future)

Reply:
I'm fine with that.

It would be nice to have a way to configure arch-specific (i.e., x86- or arm64-specific options) in the images.json, but not sure exactly how to do that.